src/PointwiseFunctions/Elasticity/Strain.hpp

```diff
@@ -8,12 +8,34 @@
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "Domain/Tags.hpp"
 #include "Elliptic/Systems/Elasticity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
 
 namespace Elasticity {
 
 /*!
- * \brief The symmetric strain \f$S_{ij}=\nabla_{(i} \xi_{j)}\f$ in the elastic
- * material.
+ * \brief The symmetric strain \f$S_{ij} = \partial_{(i} \xi_{j)}\f$ on a flat
+ * background in Cartesian coordinates.
+ */
+template <typename DataType, size_t Dim>
+void strain(gsl::not_null<tnsr::ii<DataType, Dim>*> strain,
+            const tnsr::iJ<DataType, Dim>& deriv_displacement) noexcept;
+
+/*!
+ * \brief The symmetric strain \f$S_{ij} = \nabla_{(i} \gamma_{j)k} \xi^k =
+ * \left(\partial_{(i} \gamma_{j)k} - \Gamma_{kij}\right)\xi^k\f$ on a
+ * background metric \f$\gamma_{ij}\f$.
```

Comment:
why write it this way instead of symmetrizing `\partial_i \xi_j - \Gamma^m_ij \xi_m` ?  If you keep it this way don't factor out the `\xi^k` as it this makes it look like the partial derivative is only acting on the metric

Reply:
The expression I wrote only differs from yours in that `\xi_j` is written as `\gamma_jk \xi^k`. I did that because the displacement `\xi` is a vector field and passed in to the function as such. I removed the factoring of `\xi^k`.